website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -73,6 +73,8 @@ resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
 }
 ```
 
+~> **IMPORTANT:** If setting to true for an existing cluster or imported cluster be sure to run terraform refresh after applying to enable modification of the Cloud Provider Snapshot Backup Policy going forward.
+
```

Comment:
I'm not sure if this is the best location for this - would this be for if someone is setting provider_backup_enabled = true?  If so I think including this information in the Cluster resource documentation where provider_backup_enabled argument is defined is would make more sense?

Reply:
make sense I will change it to the cluster documentation, thanks